index.js

```diff
@@ -518,8 +519,10 @@ function Argv (processArgs, cwd) {
     // command defer processing to it.
     var handlerKeys = Object.keys(self.getCommandHandlers())
     for (var i = 0, command; (command = handlerKeys[i]) !== undefined; i++) {
-      if (~argv._.indexOf(command)) {
+      if (argv._.indexOf(command) === self.level - 1) {
+        self.level += 1;
         runCommand(command, self, argv)
+        self.level -= 1;
```

Comment:
No semicolons, please. We use [standard style](https://www.npmjs.com/package/standard).